build/publish.ts

```
@@ -33,6 +33,14 @@ if (CI && (!tag || GITHUB_EVENT_NAME !== 'push')) {
         let meta;
         try {
             meta = require(`../${name}/package.json`);
+            if (tag === 'dev') {
+                try {
+                    const result = await packageJson(meta.name, { version: meta.version });
+                    if (typeof result?.version === meta.version) return progress++; // no change on dev version
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)